### DISCUSSION TITLE
Re-spacing of multi-line echo statements

### DISCUSSION THREAD

**Discussion by oleibman** [PHP-CS-Fixer/PHP-CS-Fixer#6870] [Q&A] [answered]
## Support question

My code looks this:
```php
echo "blahblahblah",
    "yaddayaddayadda",
    "etcetcetc";
```
Php-cs-fixer wants to change it to:
```php
echo "blahblahblah",
"yaddayaddayadda",
"etcetcetc";
```
I cannot figure out what rule is causing this change. Is there a way to stop it from happening?

**Comment by Wirone** [comment]
Did you try using dots `.` instead of commas `,` for concatenating strings?

**Comment by oleibman** [comment]
I hadn't. It appears that might work. Technically speaking, I think using the commas is not a concatenation, although practically speaking the they might yield the same results. I would guess that commas might be more efficient (you just need to send each string to stdout, whereas with periods you would have to concatenate first before sending the string). Also worth noting - V2 did not make this change but V3 does. At any rate, thank you for the suggestion. I would still be interested in knowing if there is a rule that controls this behavior.

**Reply by keradus** [reply]
v2 is dead for ~1.5 **years** https://github.com/PHP-CS-Fixer/PHP-CS-Fixer/releases/tag/v2.19.3 ;)

`braces` is deprecated soon [#4885]
do you see this issue with other rule, anywhere?

**Comment by Wirone** [comment, marked as answer]
The rule that is causing this change is `braces`. You can see it when you run Fixer with `--verbose` option, which prints applied fixers.

![image](https://user-images.githubusercontent.com/600668/228608289-95a0bdae-997c-4457-9c75-e6bd4adc2ff6.png)

I believe this is some kind of side effect that can be considered as a bug. I am not sure if Fixer supports commas used like this.

**Reply by oleibman** [reply]
Thanks for the answer. Now I need to think about whether to file a bug report - the behavior is certainly unexpected, and I don't think the language construct I'm using would be considered unusual.

**Reply by Wirone** [reply]
To be honest I have never seen such usage in real-life projects in my whole life, and I worked with many different projects, including legacy 🙂. I know it works like this because I saw it somewhere as a fun fact 😅. I think it can be safely considered as an edge case, but feel free to create an issue.
